src/main.js

```diff
@@ -137,19 +137,19 @@ export default class Auth {
 	}
 
 	/**
-	 * Makes sure the user is logged in and has up-to-date credentials.
-	 * @throws Will throw if the user is not signed in.
+	 * Makes sure the user is signed-in and has up-to-date credentials.
```

Comment:
Great!

But I realized, that "signed-in", "signed-out", "sign-in", "sign-out" should all be "signed in", "signed out", "sign in" and "sign out" as these are verbs. Therefore, dash there is incorrect.

Can you also fix all appearances of these verbs in the comments?